src/clj/game/cards/identities.clj

```diff
@@ -890,6 +890,38 @@
                                 (trigger-event state side :ice-subtype-changed ice)))}]
     :events {:run-ends nil}}
 
+   "Saraswati Mnemonics: Endless Exploration"
+   (letfn [(install-card [chosen]
+                        {:prompt "Select a remote server"
+                         :choices (req (conj (vec (get-remote-names state)) "New remote"))
+                         :async true
+                         :effect (req (corp-install state side (assoc chosen :advance-counter 1) target)
+                                      (effect-completed state side eid)
+                                      (let [tgtcid (:cid chosen)]
+                                        (register-turn-flag! state side
+                                                             card :can-rez
+                                                             (fn [state side card]
+                                                               (if (= (:cid card) tgtcid)
+                                                                 ((constantly false) (toast state :corp "Cannot rez due to Saraswati Mnemonics: Endless Exploration." "warning"))
+                                                                 true)))
+                                        (register-turn-flag! state side
+                                                             card :can-score
+                                                             (fn [state side card]
+                                                               (if (and (= (:cid card) tgtcid)
+                                                                        (>= (get-counters card :advancement) (or (:current-cost card) (:advancementcost card))))
+                                                                 ((constantly false) (toast state :corp "Cannot score due to Saraswati Mnemonics: Endless Exploration." "warning"))
+                                                                 true)))))})]
+   {:abilities [{:async true
+                 :label "[Click], 1 [Credits]: Install a card from HQ in a remote server, then place 1 advancement token on it. You cannot score or rez that card until your next turn begins."
```

Comment:
The `:label` won't be shown since this card only has 1 ability, but to follow our loose standard: labels should not be a word-for-word copy of the card text (which the player can already see), but a brief outline of what the ability does. Something like "Install a card from HQ" would be fine. (Also note that your `:cost` vector will auto-generate the click and credit icons on the label, if the label were shown.)